unity-renderer/Assets/DCLServices/WearablesCatalogService/LambdasWearablesCatalogService.cs

```diff
@@ -53,6 +53,50 @@ public void Dispose()
             Clear();
         }
 
+        public async UniTask<(IReadOnlyList<WearableItem> wearables, int totalAmount)> RequestOwnedWearablesAsync(
+            string userId, int pageNumber, int pageSize, CancellationToken cancellationToken, string category = null,
+            NftRarity rarity = NftRarity.None, IEnumerable<string> collectionIds = null,
+            string name = null, (NftOrderByOperation type, bool directionAscendent)? orderBy = null)
+        {
+            var queryParams = new List<(string name, string value)>
+            {
+                ("pageNumber", pageNumber.ToString()),
+                ("pageSize", pageSize.ToString()),
+            };
+
+            if (rarity != NftRarity.None)
+                queryParams.Add(("rarity", rarity.ToString().ToLower()));
+
+            if (!string.IsNullOrEmpty(category))
+                queryParams.Add(("categories", category));
+
+            if (!string.IsNullOrEmpty(name))
+                queryParams.Add(("name", name));
+
+            if (orderBy != null)
+            {
+                queryParams.Add(("orderBy", orderBy.Value.type.ToString().ToLower()));
+                queryParams.Add(("direction", orderBy.Value.directionAscendent ? "ASC" : "DESC"));
+            }
+
+            AddCollectionIdsAndCollectionCategoryParams(queryParams, collectionIds?.ToList());
+
+            // TODO: remove the hardcoded url once the lambda is deployed to the catalysts and be part of the protocol
+            (WearableWithDefinitionResponse response, bool success) = await lambdasService.Get<WearableWithDefinitionResponse>(
+                $"https://peer-ue-2.decentraland.zone/explorer-service/backpack/{userId}/wearables",
+                cancellationToken: cancellationToken,
+                urlEncodedParams: queryParams.ToArray());
+
+            if (!success)
+                throw new Exception($"The request of wearables for '{userId}' failed!");
+
+            var wearables = response.elements.Select(wd => wd.definition).ToList();
```

Comment:
This is a hotpath, remove LinQ if possible create a fixed size array (we know the length).

Reply:
Could not make it an array since `MapLambdasDataIntoWearableItem`, which is being used in many cases, requires a `List`. However i instantiated a list with a fixed element count and removed the linq conversion